CMakeLists.txt

```diff
@@ -2199,10 +2199,9 @@ elseif(APPLE)
 
   # Actually load the frameworks at the end so we don't duplicate include.
   if(SDL_FRAMEWORK_COREVIDEO)
-# FIXME: double check for video_capture
-# find_library(COREMEDIA CoreMedia)
-#    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_COREMEDIA=1")
-#    list(APPEND EXTRA_LIBS ${COREMEDIA})
+    find_library(COREMEDIA CoreMedia)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_COREMEDIA=1")
+    list(APPEND EXTRA_LIBS ${COREMEDIA})
```

Comment:
Does `COREMEDIA` expand to have `-Wl,-weak_framework,AVFoundation` too? (see changes to `configure.ac`)

Reply:
I dont know all the magic stuff with macosx and version ...
it builds correctly on my machine. it builds on the github (so it has the correct header somehow) but fails while linking